packages/frontend/src/model/model_editor.tsx

```diff
@@ -38,6 +38,7 @@ import {
 import "./model_editor.css";
 
 export default function ModelPage() {
+    throw "an error";
```

Comment:
I left this in here to demo the fix, because it matters where the error comes from. Error that happen outside of `PageContainer` will not have a working menubar.